pytest/tests/loadtest/locust/download_contracts.sh

```diff
@@ -17,3 +17,7 @@ wget https://raw.githubusercontent.com/sweatco/sweat-near/main/res/sweat.wasm -O
 wget https://raw.githubusercontent.com/sweatco/sweat-near/main/res/sweat_claim.wasm -O ${TARGET_CONTRACTS_DIR}/sweat_claim.wasm
 ln -s ${SOURCE_CONTRACTS_DIR}/fungible_token.wasm ${TARGET_CONTRACTS_DIR}/fungible_token.wasm
 ln -s ${SOURCE_CONTRACTS_DIR}/backwards_compatible_rs_contract.wasm ${TARGET_CONTRACTS_DIR}/congestion.wasm
+
+# The `inscription.near` contract has been downloaded using:
+# ./neard view-state dump-code --account-id inscription.near --output inscription.wasm
+# from mainnet at block height 123138647
```

Comment:
@tayfunelmas is that ok?
I couldn't find a link to the contract, or the source...
https://near.zulipchat.com/#narrow/stream/295302-general/topic/Source.20code.20for.20inscription.2Enear.2FNEAT.3F/near/449317056

Reply:
maybe can you make this a echo print (eg. "reusing inscriptiuon.near contract downloaded from mainnet") so the person running this gets a notification to also run this command to update it.

Reply:
Added